cronet/src/main/java/io/grpc/cronet/CronetClientTransport.java

```diff
@@ -151,13 +151,14 @@ public void run() {
     return new StartCallback().clientStream;
   }
 
-  @SuppressWarnings("GuardedBy")
   @GuardedBy("lock")
   private void startStream(CronetClientStream stream) {
-    streams.add(stream);
-    // TODO(b/145386688): This access should be guarded by 'stream.transportState().lock'; instead
-    // found: 'this.lock'
-    stream.transportState().start(streamFactory);
+    synchronized (lock) {
```

Comment:
synchronized (lock) already exists at the caller site (newStream method) and should not be necessary to add.

Reply:
I can see a query raised with Eric as below , hope as of now no action required. please confirm?
https://github.com/grpc/grpc-java/issues/6578#issuecomment-2441594655